tests/phpunit/Mutator/Sort/SpaceshipTest.php

```diff
@@ -70,5 +70,53 @@ public function mutationsProvider(): iterable
 PHP
             ,
         ];
+
+        yield 'It does not swap operators when result is identical zero on the right side' => [
+            <<<'PHP'
+<?php
+
+($a <=> $b) === 0;
+PHP
+        ];
+
+        yield 'It does not swap operators when result is equal to zero on the right side' => [
+            <<<'PHP'
+<?php
+
+($a <=> $b) == 0;
```

Comment:
How about `($a <=> $b) == 0.0;`?

Reply:
will add.

I thought about this case but assumed that people won't use floats here :)

Reply:
@sanmai done

Reply:
People do things :(